terraform/environment/prod/20-bootstrap.tf

```diff
@@ -7,9 +7,9 @@ module "bootstrap" {
   keyvault_name         = "bastion-${local.environment}-hmcts-kv"
   subnet_name           = "bastion"
   subnet_address        = local.subnet
-  vnet_address_space    = [local.subnet]
+  vnet_address_space    = [local.subnet, local.jumpbox_subnet, local.az_bastion_subnet, local.fslogix_subnet]
```

Comment:
Do you actually want 4 vnet address spaces? normally you would just extend the vnets address space and allocate subnets in that range

Reply:
Will update to follow that approach